airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1518,14 +1517,17 @@ def _bq_cast(string_field, bq_type):
     elif bq_type == 'FLOAT' or bq_type == 'TIMESTAMP':
         return float(string_field)
     elif bq_type == 'BOOLEAN':
-        assert string_field in set(['true', 'false'])
+        if string_field not in (['true', 'false']):
```

Comment:
This can be done without the braces `'vo' not in ['true', 'false']`

Reply:
done :)